src/auth0-session/config.ts

```diff
@@ -174,6 +175,19 @@ export interface SessionConfig {
    */
   name: string;
 
+  /**
+   * By default, the session is stateless and stored in an encrypted cookie. But if you want a stateful session
+   * you can provide a store with `get`, `set` and `destroy` methods to store the session on the server side.
+   */
+  store?: SessionStore<any>;
+
+  /**
+   * A function for generating a session id when using a custom session store.
+   *
+   * **IMPORTANT** You must use a suitably unique value to prevent collisions.
+   */
+  genid?: <Req = any>(req: Req) => string | Promise<string>;
```

Comment:
Nit: given that we're using camelCase, and `genid` stands for `gen id`, should it be cased `genId`?

Reply:
Yep, I used the same values as express-openid-connect (that is influenced by express-session) - but you're right, this should be camel case - will fix